www/src/data/sidebars/doc-links.yaml

```diff
@@ -61,6 +61,9 @@
             - title: Deploying to Netlify
               link: /docs/deploying-to-netlify/
               breadcrumbTitle: Netlify
+            - title: Deploying to Cloudflare Workers
```

Comment:
Could this be moved a bit further down? Other deployment solutions tend to be requested more common and the ordering of these docs tends to be with the most relevant first. Perhaps below Zeit Now, like the recipe you wrote is on the recipes page?

Reply:
Done